src/app/beer_garden/authorization.py

```diff
@@ -809,6 +809,38 @@ def _get_garden_filter(
         ):
             return None
 
+        # Filter Connection Params
+        allow_connection_params = False
+        for roles in [user.local_roles, user.upstream_roles]:
+            for role in roles:
+                if (
+                    role.permission == Permissions.GARDEN_ADMIN.name
+                    and garden.name in role.scope_gardens
+                    and _has_empty_scopes(
+                        role,
+                        [
+                            "scope_namespaces",
+                            "scope_systems",
+                            "scope_instances",
+                            "scope_versions",
+                            "scope_commands",
+                        ],
+                    )
+                ):
+                    allow_connection_params = True
+                    break
+            if allow_connection_params:
+                break
```

Comment:
It seems this break would never run.

Reply:
It would break if you are currently evaluating `local_roles`, then it skip over evaluating `upstream_roles`